tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoDataObject.java

```diff
@@ -23,7 +23,8 @@
 @MIMEResolver.ExtensionRegistration(
         displayName = "#LBL_Enso_LOADER",
         mimeType = "application/x-enso",
-        extension = {"enso"}
+        extension = {"enso"},
+        position = 79753
```

Comment:
Curious: what des this constant mean?

Reply:
It is an ordering in the list of other definitions of `MIMEResolver`s - [position() javadoc](https://bits.netbeans.org/20/javadoc/org-openide-filesystems/org/openide/filesystems/MIMEResolver.ExtensionRegistration.html#position--). The order isn't really important - there is no other resolver for `.enso` files, but the runtime complained that all the other resolvers are positioned, just the Enso one isn't.